i18n/src/main/resources/bisq_easy_af_ZA.properties

```diff
@@ -765,7 +765,8 @@ bisqEasy.tradeState.info.seller.phase2b.fiatReceivedButton=Bevestig ontvangs van
 bisqEasy.tradeState.info.seller.phase2b.tradeLogMessage={0} het die ontvangs van {1} bevestig
 
 bisqEasy.tradeState.info.seller.phase3a.fiatPaymentReceivedCheckBox=Ek het bevestig dat ek {0} ontvang het
-bisqEasy.tradeState.info.seller.phase3a.sendBtc=Stuur {0} na die koper
+bisqEasy.tradeState.info.seller.phase3a.sendBtc.prefix=Stuur
```

Comment:
No need to edit translated strings. This will be automatically done before release. Only default i18n file need to be edited.